tests/10apidoc/03events-initial.pl

```diff
@@ -78,18 +78,21 @@ sub matrix_initialsync
 
 # A useful function which keeps track of the current eventstream token and
 #   fetches new events since it
+# $room_id may be undefined, in which case it gets events for all joined rooms.
 sub GET_new_events_for
 {
-   my ( $user ) = @_;
+   my ( $user, %params ) = @_;
+
+   my %params_to_pass = ( %params,
+      from => $user->eventstream_token,
+      timeout => 500,
+   );
 
    return $user->pending_get_events //=
       do_request_json_for( $user,
          method => "GET",
          uri    => "/api/v1/events",
-         params => {
-            from    => $user->eventstream_token,
-            timeout => 500,
-         }
+         params => \%params_to_pass,
```

Comment:
This is fine, but FYI I usually do that with

    params => {
       %params,
       more => "here",
    }